src/main/java/net/dries007/tfc/world/river/RiverBlendType.java

```diff
@@ -13,9 +13,14 @@
 public enum RiverBlendType
 {
     NONE(seed -> RiverNoiseSampler.NONE),
+    BANKED(RiverNoise::banked),
+    TALL_BANKED(RiverNoise::tallBanked),
+    FLOODPLAIN(RiverNoise::floodplain),
     WIDE(RiverNoise::wide),
     CANYON(RiverNoise::canyon),
     TALL_CANYON(RiverNoise::tallCanyon),
+    TALUS(RiverNoise::talus),
+    TERRACES(RiverNoise::terraces),
     CAVE(RiverNoise::cave);
```

Comment:
If you have the energy, let's leave a short comment on each of these saying what they do